.pre-commit-config.yaml

```diff
@@ -29,3 +29,11 @@ repos:
     hooks:
     -   id: mypy
         args: [--no-strict-optional, --ignore-missing-imports]
+-   repo: local
```

Comment:
It would be nice to have a dedicated repo for these scripts so we can use it from multiple repositories => we need to make the project URL parametrizable.